content/en/docs/concepts/storage/volumes.md

```diff
@@ -529,6 +529,15 @@ See the [GlusterFS example](https://github.com/kubernetes/examples/tree/{{< para
 
 ### hostPath {#hostpath}
 
+{{< warning >}}
+There are many ways a container with unrestricted access to the host filesystem can escalate
```

Comment:
```suggestion
There are many ways a container with access to the host filesystem can escalate
```
I feel like we could elaborate further here, but I especially feel like the word "unrestricted" here could give users a false sense of security if they have done something they think of as restricting hostPath.

Reply:
Should we mention that:
- the best restriction is not allowing folk to define a `hostPath` volume at all?
- even if you restrict what host paths a Pod / entire namespace can mount, there may well be attacks / risks that require careful mitigation?

Reply:
Good recommendations. I took a stab at re-writing this. PTAL.